TestingDAL/Program.cs

```diff
@@ -14,11 +15,20 @@ static void Main(string[] args)
         {
             Database.SetInitializer(new DataInitializer());
 
-            SubjectEntities context = new SubjectEntities();
+            using (HumanRepo repository = new HumanRepo())
```

Comment:
Please consider using here `var` keyword instead of using strong typing also please consider renaming the `repository` variable to `humanRepository`.